lib/iris/tests/test_cf.py

```diff
@@ -4,152 +4,137 @@
 # See LICENSE in the root of the repository for full licensing details.
 """Test the cf module."""
 
-# import iris tests first so that some things can be initialised before importing anything else
-import iris.tests as tests  # isort:skip
-
 import contextlib
 import io
-from unittest import mock
 
 import pytest
 
 import iris
 import iris.fileformats.cf as cf
+from iris.tests import _shared_utils
 
 
-class TestCaching(tests.IrisTest):
-    def test_cached(self):
+class TestCaching:
+    def test_cached(self, mocker):
         # Make sure attribute access to the underlying netCDF4.Variable
         # is cached.
         name = "foo"
-        nc_var = mock.MagicMock()
+        nc_var = mocker.MagicMock()
         cf_var = cf.CFAncillaryDataVariable(name, nc_var)
-        self.assertEqual(nc_var.ncattrs.call_count, 1)
+        assert nc_var.ncattrs.call_count == 1
 
         # Accessing a netCDF attribute should result in no further calls
         # to nc_var.ncattrs() and the creation of an attribute on the
         # cf_var.
         # NB. Can't use hasattr() because that triggers the attribute
         # to be created!
-        self.assertTrue("coordinates" not in cf_var.__dict__)
+        assert "coordinates" not in cf_var.__dict__
         _ = cf_var.coordinates
-        self.assertEqual(nc_var.ncattrs.call_count, 1)
-        self.assertTrue("coordinates" in cf_var.__dict__)
+        assert nc_var.ncattrs.call_count == 1
+        assert "coordinates" in cf_var.__dict__
 
         # Trying again results in no change.
         _ = cf_var.coordinates
-        self.assertEqual(nc_var.ncattrs.call_count, 1)
-        self.assertTrue("coordinates" in cf_var.__dict__)
+        assert nc_var.ncattrs.call_count == 1
+        assert "coordinates" in cf_var.__dict__
 
         # Trying another attribute results in just a new attribute.
-        self.assertTrue("standard_name" not in cf_var.__dict__)
+        assert "standard_name" not in cf_var.__dict__
         _ = cf_var.standard_name
-        self.assertEqual(nc_var.ncattrs.call_count, 1)
-        self.assertTrue("standard_name" in cf_var.__dict__)
+        assert nc_var.ncattrs.call_count == 1
+        assert "standard_name" in cf_var.__dict__
 
 
-@tests.skip_data
-class TestCFReader(tests.IrisTest):
+@_shared_utils.skip_data
+class TestCFReader:
     @pytest.fixture(autouse=True)
     def set_up(self):
```

Comment:
(In 3 places)
```suggestion
    def _setup(self):
``` 

Was this intentional (i.e. if it ain't broken, I won't fix it)? Since both work, I don't really mind, it's just a consistency thing.